registers/dump/main.go

```diff
@@ -28,8 +76,19 @@ func main() {
 	}
 	log.Printf("%#v", r)
 
-	err = r.GetEntries(&dumper{})
+	g, err := grpc.Dial(*trillianLog, grpc.WithInsecure())
+	if err != nil {
+		log.Fatalf("Failed to dial Trillian Log: %v", err)
+	}
+
+	tc := trillian.NewTrillianLogClient(g)
+
+	d := &dumper{tc: tc, ctx: context.Background(), logId: *logId}
+	err = r.GetEntries(d)
```

Comment:
`GetEntries` probably wants a `Context` passing in, I'd imagine?

Reply:
Hmm. I can use one for the http fetcher, and perhaps I should. That change would rightly belong in step 1, of course.